com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/IOSBundler.java

```diff
@@ -187,15 +187,18 @@ public static List<File> getBinariesFromArchitectures(Project project, List<Plat
     private static final String SYMBOL_EXE_RELATIVE_PATH = String.format("Contents/Resources/DWARF/dmengine");
 
     public static List<File> getSymbolDirsFromArchitectures(File buildDir, List<Platform> architectures) {
+        final String[] prefixes = {"", "src" + File.separator};
```

Comment:
when we unpack zip it has src in it. 
so I check both paths, root folder and `src` folder